tests/js/spec/views/alerts/issueRuleEditor/setupAlertIntegrationButton.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import SetupAlertIntegrationButton from 'app/views/alerts/issueRuleEditor/setupAlertIntegrationButton';
+
+describe('SetupAlertIntegrationButton', function () {
+  const organization = TestStubs.Organization();
+  const project = TestStubs.Project();
+  it('renders button if no alert integrations', async function () {
+    MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/?expand=hasAlertIntegration`,
+      body: {
+        ...project,
+        hasAlertIntegrationInstalled: false,
+      },
+    });
+    const {container} = mountWithTheme(
+      <SetupAlertIntegrationButton
+        projectSlug={project.slug}
+        organization={organization}
+      />
+    );
+    await tick();
```

Comment:
ticks are from enzyme. if it isn't passing without it wrap the expect with await waitFor

Reply:
or try using getByText instead